dev/com.ibm.ws.jdbc_fat_db2/fat/src/com/ibm/ws/jdbc/fat/db2/FATSuite.java

```diff
@@ -42,7 +42,8 @@
         };
     }
 
-    static final DockerImageName db2Image = DockerImageName.parse("aguibert/db2-ssl:1.0")
+    // Updated docker image to use TLS1.2 for secure communication
+    static final DockerImageName db2Image = DockerImageName.parse("gjwatts/db2-ssl-tls12:latest")
```

Comment:
Can you tag this image with a version as well, and use that here. 
For example `gjwatts/db2-ssl-tls12:1.0`
Using latest could be unsafe since it is assumed the latest image could be changed and introduce a breaking change without any changes actually happening in the repo itself.

Reply:
Good point, thanks for catching that!  I have renamed the container and added a specific version.  It is now `gjwatts/db2-tls12:1.0` (the ssl part seemed redundant) and can be found [here](https://hub.docker.com/r/gjwatts/db2-tls12).  I also updated the code to point to the new name and reflected the fact that the container is now based on DB2 `11.5.5.0` (the old one was `11.5.0.0a`)